server_tests/package.json

```diff
@@ -16,6 +16,9 @@
     "request": "^2.55.0",
     "rsvp": "^3.0.18"
   },
+  "scripts" :{
+      "test": "mocha"
```

Comment:
You should make this point to the one in node_modules, since its guarantted to be there rather than assuming they have mocha on the path.